Console/Program.cs

```diff
@@ -23,6 +23,7 @@ static void Main(string[] args)
             {
                 UpdateLoggingLevels(options);
                 bool combine = options.Combine;
+                string date = DateTime.Now.ToString("yyyy-MM-dd");

```

Comment:
Should be dateTime with precision down to seconds or milliseconds - currently this can still hoover up files from cancelled runs on the same day